vertx-web/src/main/java/io/vertx/ext/web/handler/impl/ErrorHandlerImpl.java

```diff
@@ -95,9 +92,11 @@ private boolean answerWithKnownFailure(RoutingContext context) {
     
     if(failure instanceof HeaderTooLongException){
       // Math.random is random enough for this purpose. Will wait between 1 ms and 11ms
-      answerDelayTimer.schedule(
-          new DelayedAnswer(context, 400, "A header was too long to process"), 1 + (long)(Math.random() * 10)
+      
```

Comment:
I think we should relive this at all. We do have: https://github.com/vert-x3/vertx-web/blob/master/vertx-web/src/main/java/io/vertx/ext/web/handler/TimeoutHandler.java in order to terminate long running requests so if a user is really concerned with the time it takes to handle the header then she should use the `TimeoutHandler`.